integration/test_resources/help/agent_aws-install_ec2ssm

```diff
@@ -0,0 +1,70 @@
+This command installs the agent on all EC2 instances in an AWS account using SSM.
```

Comment:
is there anything in here about reporting on instances that don't support ssm, or that don't support ssm by default? IIRC, suse and a few other distros were 'download the ssm agent with this curl command and manually install'

Reply:
If the instance doesn't have the SSM agent running, then we'll skip it and print a warning to the customer. The idea is "best effort". It's a good idea to add a caveat for that though, done in 6ae711ea10f99a52a33f5bc82bf4619e6b81a822

That said, I'm curious how you downloaded the SSM agent without SSM? Did you do this for the lambda?

Reply:
Internally we just throw up our hands and say darn. I think the eventual plan, long since discarded by all of the other work that's come up, was to try and force people to use ami's that already either had the agent installed by default or already had ssm support (either by installing it via packer or like, requiring people use a supported distro).

Reply:
Personally, I wouldn't mind being forced to use a distro that has SSM support, but I can see how the project priority drops once you have 80% coverage 😁